lib/osf-components/addon/components/registries/partial-registration-modal/styles.scss

```diff
@@ -0,0 +1,36 @@
+.modal-title {
+    font-size: 24px;
+}
+
+.modal-button-continue {
+    font-size: 16px;
+    font-weight: bold;
+    border-color: transparent;
+    text-transform: uppercase;
+}
+
+
+.modal-button-continue:hover {
+    background: rgb(21, 165, 235);
+    border-color: transparent;
+}
+
+.modal-button-cancel {
+    font-size: 16px;
+    font-weight: bold;
+    background: #fff;
+    border-color: #e5e5e5;
+    color: rgb(21, 165, 235);
+    text-transform: uppercase;
```

Comment:
I appreciate the care that went into these two buttons, but it's better to use an un-customized `<OsfButton>` so our styles are consistent. If we choose to redesign our buttons, that'll be its own task (and should be possible to do in one place).